Makefile

```diff
@@ -68,3 +68,19 @@ endif
 
 # TODO
 # include ./test/integration-test.mk
+
+ANP_DIR ?= dependencymagnet/anp
+
+build-addon: $(ANP_DIR)/cmd/server/main.go $(ANP_DIR)/cmd/agent/main.go cmd/apiserver-proxy/main.go cmd/user-proxy/main.go cmd/cluster-proxy/main.go
+	rm -rf bin/
+	echo "Building proxy-agent"
```

Comment:
We may remove the `echo`, the make will be echo the command, so I think echo is redundancy